HST/pipeline/pipeline_query_hst_moving_targets.py

```
@@ -92,31 +92,36 @@
+                                            instruments=instruments,
+                                            start_date=start_date,
+                                            end_date=end_date,
+                                            logger=logger,
+                                            max_retries=retry)
+logger.info('List of program ids: ' + str(program_ids_list))
 
 if taskqueue:
     # If there is a missing HST_PIPELINE/hst_<nnnnn> directory, queue query-hst-products
-    for proposal_id in proposal_ids:
+    for proposal_id in program_ids_list:
         logger.info(f'Queue query_hst_products for {proposal_id}')
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)